js/internal-template-events.js

```diff
@@ -1,10 +1,11 @@
 (function ($) {
-  /*Submit url to rate form*/
-  $(document).on('submit', '#RateThisPageForm', function () {
+  function ratePage() {
     document.getElementById('url').value = window.location.href;
 
     if ($('input#website').val().length) {
       return false;
     }
-  });
+  }
+  /*Submit url to rate form*/
+  $('#RateThisPageForm').on('submit', ratePage);
```

Comment:
As discussed, the binding statements can be switched back to the $(document) way of doing things.